src/high-level/specialize-constructor.lisp

```diff
@@ -30,6 +32,7 @@
            ((subtypep type 'double-float) 'tensor/double-float)
            ((subtypep type '(complex single-float)) 'tensor/complex-single-float)
            ((subtypep type '(complex double-float)) 'tensor/complex-double-float)
+           ((subtypep type 'bit) 'tensor/bit)
```

Comment:
The amount of repetition is making be think there must be a better way. Probably in a different PR.

Reply:
Yes, that is one of the reasons I paused this.